<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulletEditorClass</class>
 <widget class="QMainWindow" name="BulletEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BulletEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Browser">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>资源浏览器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Display">
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>形态</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Editing">
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>编辑</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_openFile"/>
    <addaction name="action_saveFile"/>
    <addaction name="action_saveFileAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Objects">
   <property name="windowTitle">
    <string>对象</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Decorate">
   <property name="windowTitle">
    <string>修饰</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="action_openFile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_saveFile">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_saveFileAs">
   <property name="text">
    <string>另存为..</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>工作目录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BulletEditor.qrc"/>
 </resources>
 <connections/>
</ui>
